.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "CryptoPkg/Library/OpensslLib/openssl"]
 	path = CryptoPkg/Library/OpensslLib/openssl
 	url = https://github.com/openssl/openssl
-[submodule "SoftFloat"]
+[submodule "ArmPkg/Library/ArmSoftFloatLib/SoftFloat"]
```

Comment:
Could you elaborate on the failure mode?  Which git client are you using?

This has worked as expected since I have added it 5 years ago so I would like to understand what has changed.

Reply:
I discussed it with @Javagedes and have clarity on what happens.

Closing.